yt/utilities/answer_testing/framework.py

```diff
@@ -1040,43 +1022,6 @@ def compare(self, new_result, old_result):
                 )
 
 
-class GenericImageTest(AnswerTestingTest):
```

Comment:
@brittonsmith just wanted to check that this does *not* get used in enzo or grackle, right?

Reply:
I don't *think* we do.  But that's why I'm asking!

Reply:
Grackle doesn't use either of these classes. Enzo uses `AnswerTestingTest`, but not `GenericImagetest`.

Reply:
`AnswerTestingTest` isn't removed in this PR, so it should be good !

Reply:
@matthewturk any chance we might move forward with this soon ?